src/ontology/mondo-ingest.Makefile

```diff
@@ -553,7 +557,9 @@ $(TMPDIR)/nord.tsv:
 	wget "https://rdbdev.wpengine.com/wp-content/uploads/mondo-export/rare_export.tsv" -O $@
 
 $(EXTERNAL_CONTENT_DIR)/%.robot.owl: $(EXTERNAL_CONTENT_DIR)/%.robot.tsv
-	$(ROBOT) template --template $< \
+	$(ROBOT) template \
+		--template $< \
+		--prefix "orcid: https://orcid.org/" \
```

Comment:
Necessary so that the `orcid` is correctly transformed into a URI in OWL format.